app/pods/components/agenda/agendaitem/agendaitem-case/agendaitem-titles-edit/component.js

```diff
@@ -49,9 +52,15 @@ export default class AgendaitemTitlesEdit extends Component {
 
     try {
       await saveSubcaseTitles(this.args.agendaitem, propertiesToSetOnAgendaitem, propertiesToSetOnSubcase, shouldResetFormallyOk);
-      if (this.newsletterInfo && this.newsletterInfo.get('hasDirtyAttributes')) {
+      if (this.newsletterInfo &&
+        (this.newsletterInfo.get('hasDirtyAttributes') || this.args.agendaitem.showAsRemark)) {
+          if (this.args.agendaitem.showAsRemark) {
+            this.newsletterInfo.set('richtext', title);
+            this.newsletterInfo.set('title',shortTitle);
+          }
         await this.newsletterInfo.save();
       }
+
```

Comment:
We gaan nu regelmatiger in deze IF block komen, waardoor we in geval van mededelingen vaker de newsletterInfo gaan saven ook al zijn er mogelijk geen changes aan de titels.
Dit zou kunnen concurrency veroorzaken als iemand anders het kort bestek editeert.
Maar aangezien KB enkel werkt aan die van nota's, zouden we niet zo vaak tegen die concurrency mogen lopen.
De mededelingen worden aangepast door kanselarij

De nieuwsbrief heeft normaal nooit dirtyAttributes als we een nota aan het bekijken zijn. Want de gebruiker kan enkel de "Toon in kort bestek" aanpassen bij mededelingen wat een dirtyAttribuut zou triggeren bij newsletterInfo.

Lange verwoording, maar dus ok